config/install/core.entity_form_display.node.alert_banner.default.yml

```diff
@@ -1,19 +1,25 @@
+uuid: 230ef2cf-e13b-4b92-8009-654bfcd92679
```

Comment:
Note: Adding UUIDs to configuration files is typically handled by Drupal during export. These should be removed from install config as they can cause conflicts when the module is installed on different sites.